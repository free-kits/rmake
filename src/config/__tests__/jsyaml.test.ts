import yaml from 'js-yaml';

test('test yaml load', () => {
    const testData = `
nav:
    title: 组件
    path: /components
group:
    path: /components/base
    title: 基础组件
title: Button 按钮`;
    const data: any = yaml.safeLoad(testData);
    expect(data).toMatchSnapshot();
});
