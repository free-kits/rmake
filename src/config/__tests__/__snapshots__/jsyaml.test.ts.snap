// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test yaml load 1`] = `
Object {
  "group": Object {
    "path": "/components/base",
    "title": "基础组件",
  },
  "nav": Object {
    "path": "/components",
    "title": "组件",
  },
  "title": "Button 按钮",
}
`;
